crates/core/src/instance.rs

```diff
@@ -41,9 +40,7 @@ use std::{
     time::{Duration, Instant},
 };
 
-pub const RECV_DEFAULT_TIMEOUT_MS: Duration = Duration::from_millis(10000);
-
-pub const RETRY_VALIDATION_DURATION_MIN: Duration = Duration::from_millis(500);
+pub const RETRY_VALIDATION_DURATION_MIN: Duration = Duration::from_millis(15000);
```

Comment:
Is it significant that now the MIN is much larger than the MAX? (15000 > 3600)

Reply:
Oh.  max is in seconds, not millis.  Min is much less that max!

Reply:
whoops again!